open-reac/src/main/java/com/powsybl/openreac/parameters/input/json/OpenReacParametersDeserializer.java

```diff
@@ -39,126 +150,12 @@ public OpenReacParameters deserialize(JsonParser parser, DeserializationContext
-                }
-                case "minNominalVoltageIgnoredVoltageBounds" -> {
-                    parser.nextToken();
-                    parameters.setMinNominalVoltageIgnoredVoltageBounds(parser.getValueAsDouble());
-                }
-                case "maxPlausiblePowerLimit" -> {
-                    parser.nextToken();
-                    parameters.setPQMax(parser.getValueAsDouble());
-                }
-                case "lowActivePowerDefaultLimit" -> {
-                    parser.nextToken();
-                    parameters.setLowActivePowerDefaultLimit(parser.getValueAsDouble());
-                }
-                case "highActivePowerDefaultLimit" -> {
-                    parser.nextToken();
-                    parameters.setHighActivePowerDefaultLimit(parser.getValueAsDouble());
-                }
-                case "defaultQmaxPmaxRatio" -> {
-                    parser.nextToken();
-                    parameters.setDefaultQmaxPmaxRatio(parser.getValueAsDouble());
-                }
-                case "defaultMinimalQPRange" -> {
-                    parser.nextToken();
-                    parameters.setDefaultMinimalQPRange(parser.getValueAsDouble());
-                }
-                case "defaultVariableScalingFactor" -> {
-                    parser.nextToken();
-                    parameters.setDefaultVariableScalingFactor(parser.readValueAs(Double.class));
-                }
-                case "defaultConstraintScalingFactor" -> {
-                    parser.nextToken();
-                    parameters.setDefaultConstraintScalingFactor(parser.readValueAs(Double.class));
-                }
-                case "reactiveSlackVariableScalingFactor" -> {
-                    parser.nextToken();
-                    parameters.setReactiveSlackVariableScalingFactor(parser.readValueAs(Double.class));
-                }
-                case "twoWindingTransformerRatioVariableScalingFactor" -> {
-                    parser.nextToken();
-                    parameters.setTwoWindingTransformerRatioVariableScalingFactor(parser.readValueAs(Double.class));
-                }
-                case "shuntVariableScalingFactor" -> {
-                    parser.nextToken();
-                    parameters.setShuntVariableScalingFactor(parser.readValueAs(Double.class));
-                }
-                case "optimizationAfterRounding" -> {
-                    parser.nextToken();
-                    parameters.setOptimizationAfterRounding(parser.getValueAsBoolean());
-                }
-                default -> throw new IllegalStateException("Unexpected field: " + parser.getCurrentName());
+            BiConsumer<JsonParser, OpenReacParameters> consumer = FIELD_PROCESSORS.get(parser.currentName());
+            if (consumer == null) {
+                throw new IllegalStateException("Unexpected field: " + parser.currentName());
             }
+            consumer.accept(parser, parameters);
```

Comment:
I wonder if it won't be safer to try/catch the PowsyblException to rethrow the underlying IOException here.  
Just in case some processing is made by `StdDeserializer` general mechanism when an IOException is encountered.